### PR TITLE
fix the context display and update readme

### PR DESCRIPTION
**Pull request by solarpowerinncr**
Fix these problems:
1. The Context Search tiddler doesn't display the count of tiddler correctly
2. Modified the sidebar context tiddler respond to both sidebar search and advancedsearch
3. Add the costum Standard Search which hides context tab to plugin
4. Add description of parameter 'min-term-length' which set the minimum search length

### CODE REVIEW

File: plugins/contextPlugin/search_visualizer.tid [hunk 1]
```diff
@@ -2,7 +2,16 @@ title: $:/plugins/danielo515/ContextPlugin/visualizer
...
+\define searchResults()
+<$set name="resultCount" value="""<$count filter="[!is[system]search{$(searchTiddler)$}]"/>""">
+
+{{$:/language/Search/Matches/Standard/Matches}}
+
+</$set>
+<$list filter="[!is[system]search{$(searchTiddler)$}sort[title]limit[250]]">
   {{!!title||$:/core/ui/ListItemTemplate}}
-  <$context term={{$:/temp/search}} />
+  <$context term={{$(searchTiddler)$}} />
```
Comment (danielo515) [inline plugins/contextPlugin/search_visualizer.tid:13]: Where is this `searchTiddlers` value coming from? I can't see a set or macro referencing it.

Comment (danielo515) [reply plugins/contextPlugin/search_visualizer.tid:13]: Ok, I saw the docs from 2014. This plugin is from 2013 (wow, time flies) that's why I was not aware.

**Comment by danielo515** [comment]
Thanks @solarpowerinncr

**Comment by solarpowerinncr** [comment]
Glad to be of help. Maybe you would like to update the tiddlyspot web page as well. I recently saw many projects using this plugin, but stuck in a very old version.

**Comment by danielo515** [comment]
My plan is to delete the tiddlyspot page and use github webpages as all my other plugins. This plugin is so old that I was not technically capable of such things back on the day.

**Comment by solarpowerinncr** [comment]
Wow that would be very nice!

**Pull request merged**